sdk/cosmos/azure-cosmos-table/pom.xml

```diff
@@ -0,0 +1,17 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
One reason your pull request may be failing is because the README.md is expected to have certain headings. For now, you can copy the contents of https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/template/azure-sdk-template/README.md and fill in whatever information you can.